app/controllers/dashboards_controller.rb

```diff
@@ -54,9 +54,9 @@ def push
     if version.blank?
       flash.alert = 'No preview version available'
     else
-      production_version = Version.create(number: version.number, production: true, user: current_user)
+      version.update(production: true)
 
-      if production_version.persisted?
+      if version.persisted?
```

Comment:
This conditional will now return `true` 100% of the time since `version` is an already existing (persisted) `version`.

Reply:
I removed the condition check.